README.md

```diff
@@ -50,37 +52,62 @@ To keep a single config file and aid in running multiple tests at once, RTA read
 utils.setupEnvironmentFromConfigFile('<PATH-TO-CONFIG-FILE>')
 ```
 
- to setup the environment for you. To avoid having to do this in each test file you can setup a global include in your mocha package.json setup like:
+ to setup the environment for you. 
+ 
+ To avoid having to do this in each test file you can setup a global include in your mocha package.json setup like:
 
  ```json
 "mocha": {
-	"timeout": 5000,
-	"file": [
-		"./src/test/include.ts"
-	],
-	"require": [
-		"ts-node/register",
-		"source-map-support/register"
-	]
-}
+    "timeout": 5000,
+    "file": [
+      "./dist/tests/include.js"
+    ],
+    "require": [
+      "source-map-support/register"
+    ]
+  }
 ```
 
-be sure to change the include.ts path to match where you put your include file.
+>> Note: Change the include.ts path to match where you put your include file.
 
-If you're going to use the OnDeviceComponent then there are a number of files that need to be copied over into your app. If you're not using the [BrightScript Extension for VSCode](https://marketplace.visualstudio.com/items?itemName=celsoaf.brightscript) yet then now is a great time to try it out. If you are all you need to is add:
+If you're going to use the OnDeviceComponent then there are a number of files that need to be copied over into your app. If you're not using the [BrightScript Extension for VSCode](https://marketplace.visualstudio.com/items?itemName=celsoaf.brightscript) yet then now is a great time to try it out. 
+If you are all you need is to add the following to your bsconfig.json:
```

Comment:
Reading this, it needs some grammar fix. I think I was trying to say `If you did everything correct, all you need to do is add the following to your bsconfig.json:` Can you update this since you're already modifying this?